server/neptune/workflows/internal/activities/terraform.go

```diff
@@ -1,16 +1,66 @@
 package activities
 
-import "context"
+import (
+	"context"
+	"strings"
 
-type terraformActivities struct{}
+	"github.com/hashicorp/go-version"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/events/terraform/ansi"
+	"github.com/runatlantis/atlantis/server/neptune/terraform"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/job"
+)
 
-// Terraform Init
+type terraformClient interface {
+	RunCommand(ctx context.Context, jobID string, path string, args []string, customEnvVars map[string]string, v *version.Version) <-chan terraform.Line
+}
+
+type terraformActivities struct {
+	TerraformClient  terraformClient
+	DefaultTFVersion *version.Version
+}
 
+// Terraform Init
 type TerraformInitRequest struct {
+	Step      job.Step
+	Envs      map[string]string
+	JobID     string
+	TfVersion string
+	Path      string
 }
 
-func (t *terraformActivities) TerraformInit(ctx context.Context, request TerraformInitRequest) error {
-	return nil
+type TerraformInitResponse struct {
+	Output string
+}
+
+func (t *terraformActivities) TerraformInit(ctx context.Context, request TerraformInitRequest) (TerraformInitResponse, error) {
+	// Resolve the tf version to be used for this operation
+	tfVersion, err := t.resolveVersion(request.TfVersion)
+	if err != nil {
+		return TerraformInitResponse{}, err
+	}
+
+	// Build tf command
+	terraformInitCmd := terraform.CommandArguments{
```

Comment:
```suggestion
	cmd := terraform.CommandArguments{
```

we already know it's terraform init based on the function, keep var names concise